Dockerfile

```diff
@@ -22,6 +22,10 @@ RUN rm -rf /etc/localtime \
     && dpkg-reconfigure -f noninteractive tzdata
 
 WORKDIR /build
+COPY go.mod .
+COPY go.sum .
+RUN GOPROXY=https://goproxy.cn,direct go mod download
```

Comment:
Is it better to add parameters to make this step optional?

Reply:
You mean the go proxy?